raiden-dapp/src/views/DisclaimerRoute.vue

```diff
@@ -14,23 +14,26 @@
           >
             <li>
               {{ bulletPoint.text }}
-              <a :href="disclaimerUrlMappings[bulletPoint.url]" target="_blank">
+              <a :href="disclaimerUrlMappings[index]" target="_blank">
```

Comment:
This is not the index anymore. Rather the `name` or something similar. :shrug: 
But the more important is that you can resolve [this](https://github.com/raiden-network/light-client/pull/2531#discussion_r566167769) now. Before both `v-for` where using the same numbers (indexes) as keys. Now they do no-more. So you can get rid of the `<div>` which is potential layout risk. If it should be still an issue I would recommend to at least use a `<p>` as this is an inline element. I don't know what "magic" Vuetify apply automatically else.

Reply:
Done!